docs/reference/release-notes/7.0.0-alpha2.asciidoc

```diff
@@ -18,15 +18,15 @@ Index APIs::
 * Always enforce cluster-wide shard limit {pull}34892[#34892] (issues: {issue}20705[#20705], {issue}34021[#34021])
 
 Ranking::
-* Forbid negative scores in functon_score query {pull}35709[#35709] (issue: {issue}33309[#33309])
+* Forbid negative scores in function_score query {pull}35709[#35709] (issue: {issue}33309[#33309])
```

Comment:
I understand this is a release notes file, i'm happy to drop the file if that's requested